orbit-2-core/src/test/java/com/babylon/orbit2/ErrorHandlingTest.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2020 Babylon Partners Limited
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.babylon.orbit2
+
+internal class ErrorHandlingTest
```

Comment:
missing content

Reply:
Yup this is just so I don't forget to add it later :)

Reply:
But I guess I can remove for now and create a ticket instead